app/assets/stylesheets/custom.css.scss

```diff
@@ -0,0 +1,62 @@
+body {
+  background: image-url('confectionary.png') repeat;
+  background-attachment: fixed;
+}
+
+.logo {
+  margin-right: 10px;
+}
+
+.navbar {
+  margin-right: 23px;
+  margin-top: 35px;
+}
+
+#item-title {
+  font-weight: bold;
+  margin-top: 20px;
+  margin-left: 30px;
+  text-shadow: 0px 2px 3px #666;
```

Comment:
Color literals like `#666` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>`0px` should be written without units as `0`